payroll/charts.py

```diff
@@ -12,17 +10,13 @@ def _get_bar_color(self, lower_edge, upper_edge):
 
     def bin_salary_data(self, data):
         float_data = np.asarray(data, dtype='float')
-
-        # Size of the bins
-        multiplier = 25000
-
-        # This is to make the appropriate number of bins
         max_value = np.amax(float_data)
-        bin_num = math.ceil(max_value / multiplier)  # rounding up to capture max value
-        bin_edges = np.array([], dtype='float')
 
-        for i in range(bin_num + 1):  # adding 1 to get appropriate number of bins
-            bin_edges = np.append(bin_edges, i * multiplier)
+        bin_size = 10000
```

Comment:
seems like one or both of these would be good candidates for `chart_settings`.

Reply:
Any reason for not just sticking it in there?

Reply:
didn't feel strongly one way or the other, tbh. do you?